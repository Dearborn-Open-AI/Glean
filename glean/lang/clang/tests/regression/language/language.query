query: src.FileLanguage _
transform: [gensort, []]
