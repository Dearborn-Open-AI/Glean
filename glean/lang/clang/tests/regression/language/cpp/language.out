[
  "@generated",
  { "key": { "file": { "key": "test.cpp" }, "language": 2 } },
  { "key": { "file": { "key": "test.h" }, "language": 2 } }
]